src/app/pages/TokenDashboardPage/hook.ts

```diff
@@ -35,7 +35,7 @@ export const useTokenInfo = (scope: SearchScope, address: string, enabled = true
   }
 }
 
-export const useTokenTransfers = (scope: SearchScope, address: string) => {
+export const useTokenTransfers = (scope: SearchScope, address: string, nftId?: string) => {
```

Comment:
Do we prefer to keep useTokenTransfers intact and add `useInstanceTokenTransfers` or we are fine with ternary operators?

Reply:
I think a good function signature would be
```ts
export const useTokenTransfers = (scope: SearchScope, params: { account: string } | { nft_id: string, contract_address: string }) => {
```

Reply:
hm, nope, that apparently permits `useTokenTransfers(scope, { account: '4', contract_address: 'a' })`

Reply:
updated